### ISSUE TITLE
kubesec not running on alpine linux

### ISSUE THREAD

**Issue by neerfri** [shyiko/kubesec#10]
# Problem

Trying to run `kubesec` in alpine linux results in:
```
bash-4.4# ./kubesec
bash: ./kubesec: No such file or directory
```

# How to reproduce

```
$ docker build -t kubesec_bug https://gist.githubusercontent.com/neerfri/08b3f87a6ad9f5bdb9bf771616175407/raw/e8b8d578122f499bfbb63a8c49811d25b159e64b/Dockerfile
$ docker run --rm -it kubesec_bug /bin/sh -c '/root/kubesec'
$ docker run --rm -it kubesec_bug /bin/sh -c 'ldd /root/kubesec'
```

# Notes

I believe the reason is that it's dynamically linked against libraries (glibc) that don't exist in alpine (by default).
You can see a similar example and suggestions for solutions in:
https://github.com/gliderlabs/docker-alpine/issues/78
and https://github.com/blang/golang-alpine-docker

**Comment by shyiko** [comment]
Fixed in [0.9.0](https://github.com/shyiko/kubesec/blob/master/CHANGELOG.md#090---2018-06-25).

**Comment by neerfri** [comment]
@shyiko Thanks! You rock! 🤘
